src/ReleaseHistory.md

```diff
@@ -1,5 +1,8 @@
 # Microsoft Json Schema Packages
 
+## **Unreleased**
+* FEATURE: Add new option `--generateintegeras = int | long | biginteger | auto` and default as `int`. [#158](https://github.com/microsoft/jschema/pull/158)
```

Comment:
>generateintegeras [](http://example.com/codeflow?start=29&length=17)

Please include the dashes as they appear in `Options.cs`, this should look something like `--generate-integer-as` to avoid confusion and help with readability.

Reply:
yes! fixed.